bool Standstill
bool StartReferenceMeasure
bool ResetReferenceDistance
bool StartReferenceMeasureConfirmation
bool ResetReferenceDistanceConfirmation
float64 ReferenceMeasure
float64 TotalDistanceKM